gdcdictionary/python/__init__.py

```diff
@@ -73,7 +73,12 @@ def load_yaml(self, name):
                 except Exception as e:
                     self.logger.error("Error in file: {}".format(name))
                     raise e
-            return yaml.safe_load(f)
+            try:
+                return yaml.load(f, Loader=yaml.CSafeLoader)
+            except AttributeError as e:
+                self.logger.warning(e)
+                self.logger.warning("To enable CSafeLoader install libyaml. Falling back to yaml.safe_load()")
+                return yaml.safe_load(f)
```

Comment:
pyyaml sets a [flag](https://github.com/yaml/pyyaml/blob/0abad85a17ba75c0fb431feea7a6a06125341a99/lib/yaml/__init__.py#L12-L16
) to indicate if libyaml extension is available.  let's use it:
```suggestion
            if yaml.__with_libyaml__:
                return yaml.load(f, Loader=yaml.CSafeLoader)
            else:
                return yaml.safe_load(f)
```